flink-libraries/flink-table/src/test/scala/org/apache/flink/api/scala/batch/sql/AggregationsITCase.scala

```diff
@@ -258,4 +258,22 @@ class AggregationsITCase(
     // must fail. grouping sets are not supported
     tEnv.sql(sqlQuery).toDataSet[Row]
   }
+
+  @Test
+  def testSqrtOfAggregatedSet(): Unit = {
```

Comment:
I think we do not need this ITCase. 
The `PowerExpressionTest` should cover all cases.